containers/ecr-viewer/src/app/page.tsx

```diff
@@ -56,11 +55,13 @@ const HomePage = async ({
   const allConditions = await getAllConditions();
 
   return (
-    <div className="display-flex flex-column height-viewport">
-      <Header />
+    <div className="display-flex flex-column height-viewport-header-footer">
```

Comment:
same thing here with "do we still need this div"

Reply:
Yep, controls sizing of the page content and keeps it from shrinking looks like

Reply:
Though maybe the classes could be moved down, hmm, will look